tests/yam/agama/agama_cli.pm

```diff
@@ -0,0 +1,70 @@
+## Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Check Agama CLI commands performing tests on each command and running installation.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use testapi;
+use power_action_utils qw(power_action);
+use Utils::Architectures qw(is_s390x);
+use Utils::Backends qw(is_svirt);
+
+sub agama_config_edit {
+    my $regex = shift;
+
+    script_run('agama config edit', timeout => 0);
+    wait_still_screen();
+    type_string($regex);
+    type_string(":wq\n");
+    wait_still_screen();
+}
+
+sub run {
+    my $self = shift;
+
+    my $product_id = get_var('AGAMA_PRODUCT_ID');
+    my $agama_help = script_output('agama 2>&1 | tee', proceed_on_failure => 1);
```

Comment:
would be more concise to use https://open.qa/api/testapi/#_validate_script_output in a single command.